src/features/vn/queries/vnInfo.ts

```diff
@@ -3,21 +3,19 @@ import { useQuery } from '@tanstack/react-query';
 import { VnInfoDtoSchema } from 'src/api/dtos/VnInfoDto';
 import { VnInfoMapper } from 'src/api/mappers/VnInfoMapper';
 import type { VnInfo } from 'src/api/models/VnInfo';
+import { VnService } from 'src/api/services/vnService';
 
 /**
  * Get vn info.
  * @param id Vn id.
  */
-export const getVnInfo = async(id: number): Promise<VnInfo> => {
+export const getVnInfo = async(id: string): Promise<VnInfo> => {
```

Comment:
Lets create alias type for `Id`. I believe all entities from vndb have id in format `{letter}{number}`, hence it's ok to use singe id type for all of them.